o3po/includes/class-o3po-primary-publication-type.php

```diff
@@ -1455,7 +1457,12 @@ public function get_the_content( $content ) {
             $content .= '<table class="meta-data-table">';
             $content .= '<tr><td>Published:</td><td>' . esc_html($this->get_formated_date_published( $post_id )) .  ', ' . $this->get_formated_volume_html($post_id) . ', page ' . esc_html(get_post_meta( $post_id, $post_type . '_pages', true )) . '</td></tr>';
             $content .= '<tr><td>Eprint:</td><td><a href="' . esc_attr($settings->get_plugin_option('arxiv_url_abs_prefix') . get_post_meta( $post_id, $post_type . '_eprint', true ) ) . '">arXiv:' . esc_html(get_post_meta( $post_id, $post_type . '_eprint', true )) . '</a></td></tr>';
-            $content .= '<tr><td>Scirate:</td><td><a href="' . esc_attr($settings->get_plugin_option('scirate_url_abs_prefix') . get_post_meta( $post_id, $post_type . '_eprint', true ) ) . '">' . esc_html($settings->get_plugin_option('scirate_url_abs_prefix') . get_post_meta( $post_id, $post_type . '_eprint', true )) . '</a></td></tr>';
+
+            $content .= $this->scirate_content(
+                           $settings->get_plugin_option('scirate_url_abs_prefix'), 
+                           $post_id,
+                           $post_type);
+
```

Comment:
Creating a separate function here seems overkill to me. Can we not just add a `if(!empty($scirate_url_abs_prefix))` before the already existing `$content .= ...` line?